microproxy/protocol/tls.py

```diff
@@ -74,16 +74,13 @@ def create_dest_sslcontext(insecure=False, trusted_ca_certs="", alpn=None):
     ssl_ctx = create_basic_sslcontext()
 
     if not insecure:
+        verify_mode = SSL.VERIFY_PEER | SSL.VERIFY_FAIL_IF_NO_PEER_CERT
         trusted_ca_certs = trusted_ca_certs or certifi.where()
         ssl_ctx.load_verify_locations(trusted_ca_certs)
-        ssl_ctx.set_verify(
-            SSL.VERIFY_PEER | SSL.VERIFY_FAIL_IF_NO_PEER_CERT,
-            certificate_verify_cb)
     else:
-        def verify_cb_always_true(*args, **kwargs):
-            return True
-        ssl_ctx.set_verify(SSL.VERIFY_NONE, verify_cb_always_true)
+        verify_mode = SSL.VERIFY_NONE
 
+    ssl_ctx.set_verify(verify_mode, certificate_verify_cb)
```

Comment:
What I mean is removing verify_cb_always_true is ok
But no need to create a new variable verify_mode for this.